internal/app/handlers/url.go

```diff
@@ -0,0 +1,67 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/go-chi/chi"
+	"io"
+	"net/http"
+)
+
+func (h *Handler) postHandler(w http.ResponseWriter, r *http.Request) {
+	//читаем тело запроса
+	body, err := io.ReadAll(r.Body)
+	defer r.Body.Close()
+	if err != nil {
+		http.Error(w, err.Error(), 400)
+		h.log.Errorf("unable to read request body, err: %s", err)
+
+		return
+	}
+
```

Comment:
```suggestion
defer r.Body.Close()
```

https://t.me/bushigo/21